includes/definitions/ge-digitalenergy.yaml

```diff
@@ -0,0 +1,16 @@
+os: ge-digitalenergy
+text: 'GE Digital Energy UPS'
+group: ups
+type: power
+icon: ge
+rfc1628_compat: 1
+over:
+    - { graph: device_current, text: Current }
+    - { graph: device_voltage, text: Voltage }
+    - { graph: device_load, text: Load }
+discovery:
+    -
+        sysObjectId:
+            - .1.3.6.1.4.1.818.1.100.1.1
+        sysDescr_regex:
```

Comment:
Do you need this as well? Using just sysObjectId on it's own is the preferred way unless this OID conflicts with another OS